BigriverBookstore-apple/Model/Author.swift

```diff
@@ -0,0 +1,9 @@
+//
+//  Author.swift
+//  BigriverBookstore-apple
+//
+//  Created by Viorel Porumbescu on 04/03/2019.
+//  Copyright © 2019 AgileFreaks. All rights reserved.
+//
+
+import Foundation
```

Comment:
@c-Viorel This class is not implemented/used yet? If this is the case, let's exclude it from the pr for the moment and add it on the next pr's.